File: flutter_baidu_yingyan_trace-1.0.0/lib/src/model/entity/add_entity_result.dart
import 'package:flutter_baidu_yingyan_trace/src/model/base_result.dart';

/// 创建Entity响应结果
class AddEntityResult extends BaseResult {
  AddEntityResult({required int tag, required int status, required String message})
      : super(tag: tag, status: status, message: message);

  /// map => AddEntityResult
  AddEntityResult.fromMap(Map map)
      : assert(map != null,
            'Construct a AddEntityResult，The parameter map cannot be null !'),
        super.fromMap(map);

  @override
  Map toMap() {
    return super.toMap();
  }

  @override
  String toString() {
    return "AddEntityResult{tag = $tag; status = $status; message = $message}";
  }
}

File: flutter_baidu_yingyan_trace-1.0.0/android/src/main/java/com/baidu/flutter/trace/model/entity/AddEntityResult.java
package com.baidu.flutter.trace.model.entity;

import com.baidu.flutter.trace.model.BaseResult;

/**
 * 创建Entity响应结果
 *
 * @author baidu
 */
public final class AddEntityResult extends BaseResult {

    /**
     * @param tag     响应标识（与请求标识一致）
     * @param status  状态码
     * @param message 状态信息
     */
    public AddEntityResult(int tag, int status, String message) {
        super(tag, status, message);
    }

    @Override
    public String toString() {
        return "AddEntityResult [tag=" + tag + ", status=" + status + ", message=" + message + "]";
    }

}
